--- c/include/dynamixel_sdk/port_handler.h 2023-01-01 12:00:00.000000 +0000
+++ c/include/dynamixel_sdk/port_handler.h 2023-01-01 12:00:00.000000 +0000
@@ -45,6 +45,6 @@
 static const int DEFAULT_BAUDRATE = 57600;

-int     g_used_port_num;
-uint8_t    *g_is_using;
+extern int     g_used_port_num;
+extern uint8_t    *g_is_using;

 WINDECLSPEC int     portHandler             (const char *port_name);
