--- c/src/dynamixel_sdk/port_handler.c 2023-01-01 12:00:00.000000 +0000
+++ c/src/dynamixel_sdk/port_handler.c 2023-01-01 12:00:00.000000 +0000
@@ -2,6 +2,9 @@
 #include "port_handler.h"
 #include "port_handler_linux.h"

+int     g_used_port_num;
+uint8_t    *g_is_using;
+
 int     portHandler         (const char *port_name) { return portHandlerLinux(port_name); }

 uint8_t openPort            (int port_num) { return openPortLinux(port_num); }
